src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -87,16 +87,22 @@ public FileDeps(InternalProjectConfiguration config) {
     /**
      * Returns the path to the Java SDK static libraries for this configuration. The path is cached on the provided
      * configuration. If no custom directory has been set in the project configuration and <code>useGraalPath</code>
-     * is set to <code>true</code>, it will use the <code>lib</code> directory inside the configured Graal path. If
-     * <code>useGraalPath</code> is set to <code>false</code>, a custom Java SDK will be retrieved.
+     * is set to <code>true</code>, it will use the <code>lib/static/$target-$arch/</code> directory inside the
+     * configured Graal path. If <code>useGraalPath</code> is set to <code>false</code>, a custom Java SDK will
+     * be downloaded.
      *
      * @param useGraalPath specifies if the default Java SDK path should resolve to the Graal installation dir
      * @return the location of the static libraries of the Java SDK for the arch-os for this configuration
      * @throws IOException in case anything goes wrong.
      */
     public Path getJavaSDKLibsPath(boolean useGraalPath) throws IOException {
```

Comment:
need to return 2 paths here (the old and the new), so this is List<Path>

Reply:
Wouldn't it be better to just check for the existence of the "lib/static" directory? If it does, return that, otherwise return the old path?

Reply:
For now I'm ok with that, but in general (different issue), I feel we have to add more safety to the paths we link from (more paths in a specific order, without specifying logic)